x/denommetadata/types/params.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	KeyAllowedAddresses     = []byte("AllowedAddresses")
+	DefaultAllowedAddresses = []string(nil) // no one allowed
```

Comment:
```suggestion
	DefaultAllowedAddresses = []string // no one allowed
```

Reply:
[]string (type) is not an expressioncompiler[NotAnExpr](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#NotAnExpr) will spark this error if I commit as your suggest

Reply:
ah interesting!